### PR TITLE
Add Cluster Registry to glossary page.

### PR DESCRIPTION
**Pull request by McCoyAle**
Added this term due to the work on "Federation". Associated with #5993

### CODE REVIEW

File: content/en/docs/reference/glossary/cluster-registry.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+title: Cluster Registry
+<<<<<<< HEAD:content/en/docs/reference/glossary/cluster-registry.md
```
Comment (sftim) [inline content/en/docs/reference/glossary/cluster-registry.md]: This looks like an unfinished merge. I recommend fixing that and squashing / rebasing.

**Comment by Bradamant3** [comment]
@McCoyAle -- please fix incomplete merge. Also a question: on its own this entry seems fine, but does anything else besides federation depend on cluster registry? If not, let's close and reopen or open a new one when federation and its docs are readier (see also PR #13520 ). Thanks!

**Comment by Rajakavitha1** [comment]
@McCoyAle -- please fix incomplete merge.

**Comment by sftim** [comment]
The deploy preview doesn't show an entry for Cluster Registry
(I haven't looked into why that is)

**Pull request closed without merging**